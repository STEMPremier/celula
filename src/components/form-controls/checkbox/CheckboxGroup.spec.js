/* eslint-disable react/jsx-filename-extension */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { CheckboxGroup, Checkbox } from '.';

describe('<CheckboxGroup />', () => {
  describe('render()', () => {
    test('render the CheckboxGroup component', () => {
      const wrapper = shallow(
        <CheckboxGroup
          label="CheckboxGroup Test Label"
          name="CheckboxGroup Test Name"
          handleChangeGroup={() => {}}
        >
          <Checkbox
            label="Checkbox Test Label 1"
            value="Checkbox Value Here 1"
          />
          <Checkbox
            label="Checkbox Test Label 2"
            value="Checkbox Value Here 2"
          />
        </CheckboxGroup>,
      );

      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
/* eslint-enable react/jsx-filename-extension */
