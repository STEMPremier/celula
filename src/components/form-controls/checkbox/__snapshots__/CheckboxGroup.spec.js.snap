// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CheckboxGroup /> render() render the CheckboxGroup component 1`] = `
<FormControlGroup
  className="ce-checkbox-group"
  disabled={false}
  errorClass="ce-checkbox-group--error__text"
  errorMsg=""
  formId=""
  handleChange={[Function]}
  label="CheckboxGroup Test Label"
  name="CheckboxGroup Test Name"
  selectedValues={Array []}
>
  <Checkbox
    checked={false}
    className=""
    disabled={false}
    errorMsg=""
    formId=""
    handleChange={[Function]}
    isA="checkbox"
    label="Checkbox Test Label 1"
    name=""
    value="Checkbox Value Here 1"
  />
  <Checkbox
    checked={false}
    className=""
    disabled={false}
    errorMsg=""
    formId=""
    handleChange={[Function]}
    isA="checkbox"
    label="Checkbox Test Label 2"
    name=""
    value="Checkbox Value Here 2"
  />
</FormControlGroup>
`;
